www/docs/en/dev/guide/overview/project-structure.md

```diff
@@ -0,0 +1,105 @@
+---
+license: >
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+title: Project Structure of a Cordova Project
+toc_title: Project Structure
+description: 
+---
+
+# Project Structure
+
+## CLI project directory structure
+
+Cordova CLI works with the following directory structure:
+
+```
+myapp/
+|-- package.json
+|-- config.xml
+|-- hooks/
```

Comment:
As with merges, usage of the `hooks` folder is deprecated.

Reply:
Hooks are still supported via config files though, and then can be in random locations, right?

So this here should be removed, but maybe a new paragraph that links to the hooks documentation (that hopyfully exists).

Reply:
Yes. Hooks can live anywhere, and we want them explicitly listed in config.xml now instead of auto-detected through the hooks folder.

Reply:
For later reference: https://cordova.apache.org/docs/en/9.x/guide/appdev/hooks/index.html is the docs about hooks

Reply:
I applied changes so this can be re-reviewed.